app/controllers/admin/auction_mark_payments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Admin::AuctionMarkPaymentsController < Admin::BaseController
+  def update
+    @auction = Auction.find(params[:id])
+
+    paid_auction = MarkPaidAuction.new(
```

Comment:
I am not opposed to having this service class, but since it does so little and we are just using it in one place, thoughts on marking as paid and sending the email in this controller?